src/components/project-card.tsx

```
@@ -0,0 +1,80 @@
+  raised: number
+  daysLeft: number
+  category: string
+  publisher: {
+    name: string
+    trustScore: number
+  }
+}
+
+export function ProjectCard({ project }: { project: Project }) {
+  const progress = (project.raised / project.goal) * 100
+  const Icon = categoryIcons[project.category as keyof typeof categoryIcons]
```

ESLint

[error] Missing semicolon.

(style/semi)